packages/pyolite-kernel/src/worker.ts

```diff
@@ -1,136 +1,12 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
 
+import type Pyodide from 'pyodide';
+
 import { DriveFS } from '@jupyterlite/contents';
 
 import { IPyoliteWorkerKernel } from './tokens';
 
-// TODO Once this https://github.com/pyodide/pyodide/pull/2582/files is released
-// Remove this shameless copy
-const ERRNO_CODES = {
-  E2BIG: 1,
-  EACCES: 2,
-  EADDRINUSE: 3,
-  EADDRNOTAVAIL: 4,
-  EADV: 122,
-  EAFNOSUPPORT: 5,
-  EAGAIN: 6,
-  EALREADY: 7,
```

Comment:
![](https://media.giphy.com/media/3NtY188QaxDdC/giphy.gif)

Reply:
love a good meme, but what's your thoughts here? I don't see an easy way to get these (typed or otherwise) other than (at present) the pyodide declarations. We can keep them duplicated, of course.

Reply:
Those ERRNO_CODE were not defined in `Pyodide<0.21.0`, it has been fixed in https://github.com/pyodide/pyodide/pull/2582. I really didn't like having those ERRNO_CODES values hardcoded here.

I guess we can assume they are defined now.